extension/lib/logger.js

```diff
@@ -5,8 +5,7 @@
 const { components, Cc, Ci, Cu } = require("chrome");
 const unload = require("sdk/system/unload");
 
-Cu.import('resource://gre/modules/NetUtil.jsm');
-Cu.import('resource://gre/modules/Services.jsm');
+const {TextEncoder, OS} = Cu.import("resource://gre/modules/osfile.jsm", {});
 
 const PERMS_DIRECTORY = parseInt("0755", 8);
 const PERMS_FILE = parseInt("0655", 8);
```

Comment:
What exactly? The PERMS_ constants? If that is the case please get them removed.